validator/keymanager/remote/keymanager.go

```diff
@@ -137,7 +137,9 @@ func UnmarshalOptionsFile(r io.ReadCloser) (*KeymanagerOpts, error) {
 			log.Errorf("Could not close keymanager config file: %v", err)
 		}
 	}()
-	opts := &KeymanagerOpts{}
+	opts := &KeymanagerOpts{
+		RemoteCertificate: &CertificateConfig{RequireTls: true},
```

Comment:
Wont this still fail ? All the older remote wallets will not have tls configured for

Reply:
Discussed offline, this should unblock our older users.